enabled setup of Analysis with webSDA usecse, as this is logically be…

StefanGIT opened the pull request:
…fore the next use case, it is put in front of it

This was discussed with Alexander Dietz. This is a general description on the use case currently still disabled in the molecular level. I will add this use case also today. The guide book also stands by its own. It goes into details how to create analysis like in the live paper https://humanbrainproject.github.io/hbp-bsp-live-papers/2019/bruce_et_al_2019/bruce_et_al_2019.html

Alexander also asked somebody who would be able to evaluate and test this, for this case we would suggest: (... mail address will be send by mail)

alex4200 commented:
@StefanGIT Is it correct you want to remove the disabled use case "Set up a Brownian dynamics simulation for calculating protein-protein association rate constants" from the usecases.json? 

Also, can you please add the contributors to the new use case (please see other examples)?

Thanks

StefanGIT commented:
Yes, the use case "Set up a Brownian dynamics simulation for calculating protein-protein association rate constants" is replaced by the one i moved to the top (as it is logically before the Analysis of simulation results).


I added the contributors and a mail address we use in the lab for contacting. I slightly modified the title on Rebecca's suggestion again to: Calculate protein-protein bimolecular association rate constants with webSDA


Stefan


Von: alex4200 <notifications@github.com>
Gesendet: Donnerstag, 7. Mai 2020 11:21
An: antonelepfl/usecases
Cc: Richter, Stefan; Mention
Betreff: Re: [antonelepfl/usecases] enabled setup of Analysis with webSDA usecse, as this is logically be… (#349)


@StefanGIT<https://github.com/StefanGIT> Is it correct you want to remove the disabled use case "Set up a Brownian dynamics simulation for calculating protein-protein association rate constants" from the usecases.json?

Also, can you please add the contributors to the new use case (please see other examples)?

Thanks

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/antonelepfl/usecases/pull/349#issuecomment-625136165>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABRB7RVORRREJP4XSLT2JFTRQJ4QVANCNFSM4M3EVOMA>.

antonelepfl commented on src/assets/config_files/usecases.json:
```diff
@@ -1,6 +1,32 @@
-[
+f[
```
Could you please remove this "f"?

StefanGIT commented:
?done.



Molecular and Cellular Modeling Group (MCM)
HITS gGmbH
Schloss-Wolfsbrunnenweg 35
69118 Heidelberg
Germany

phone: +49 - 6221 - 533 279
fax: +49 6221 533298
email: Stefan.Richter@h-its.org
http://www.h-its.org
_________________________________________________
Amtsgericht Mannheim / HRB 337446
Managing Directors:
Dr. Gesa Schönberger
________________________________
Von: Stefano Antonel <notifications@github.com>
Gesendet: Montag, 25. Mai 2020 10:22
An: antonelepfl/usecases
Cc: Richter, Stefan; Mention
Betreff: Re: [antonelepfl/usecases] enabled setup of Analysis with webSDA usecse, as this is logically be… (#349)


@antonelepfl commented on this pull request.

________________________________

In src/assets/config_files/usecases.json<https://github.com/antonelepfl/usecases/pull/349#discussion_r429800449>:

> @@ -1,6 +1,32 @@
-[
+f[


Could you please remove this "f"?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/antonelepfl/usecases/pull/349#pullrequestreview-417542592>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABRB7RTEN2ZO6LTZOF2FLLLRTITEFANCNFSM4M3EVOMA>.

antonelepfl commented:
Hi Stefan,
Is the idea of this use case to open directly your web app (https://websda.h-its.org/) or add in a target Collab an entry with your app embedded (like what Luca is doing with [Feature Extraction](https://collab.humanbrainproject.eu/#/collab/1655/nav/66850))

alex4200 commented:
@StefanGIT I suggest to change the information in the usecases.json so that the use case link (https://nest-desktop.apps.hbp.eu/#/) will be directly opened in a new tab, and to remove the intermediate Collab. 

What do you think?

antonelepfl commented:
@StefanGIT If you agree to open directly the webapp I can handle the changes on the usecases.json

StefanGIT commented:
Yes, the webapp can directly opened.


Thanks,


Stefan



_________________________________________________
Amtsgericht Mannheim / HRB 337446
Managing Directors:
Dr. Gesa Schönberger
________________________________
Von: Stefano Antonel <notifications@github.com>
Gesendet: Montag, 25. Mai 2020 11:59
An: antonelepfl/usecases
Cc: Richter, Stefan; Mention
Betreff: Re: [antonelepfl/usecases] enabled setup of Analysis with webSDA usecse, as this is logically be… (#349)


@StefanGIT<https://github.com/StefanGIT> If you agree to open directly the webapp I can handle the changes on the usecases.json

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://github.com/antonelepfl/usecases/pull/349#issuecomment-633490370>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ABRB7RROSFP2H7EW54TCO5TRTI6PDANCNFSM4M3EVOMA>.
